### PR TITLE
Updated software citations in documentation (#1157)

### LINKED ISSUE #1157: Update software citations in documentation
The SciPy and Astropy citations in https://github.com/poliastro/poliastro/blob/main/docs/source/bibliography.md are outdated, we should use the most recent ones:

- https://www.nature.com/articles/s41592-019-0686-2
- https://iopscience.iop.org/article/10.3847/1538-3881/aabc4f

### PR DESCRIPTION
**Pull request by itsabhianant**
Fixes #1157

### CODE REVIEW

**Comment by itsabhianant** [comment]
I have updated the software citations as per Issue #1157 . Please do let me know if any changes need to be done. 

Best,
Abhishek

File: docs/source/bibliography.md [hunk 1]
```diff
@@ -52,4 +50,4 @@ IDEs and browsers for my whole academic life.
   Droettboom, Erik Bray, Tom Aldcroft, Matt Davis et al. \"Astropy: A
   community Python package for astronomy.\" *Astronomy & Astrophysics*
   558 (2013): A33.
-  [DOI:10.1051/0004-6361/201322068](http://dx.doi.org/10.1051/0004-6361/201322068)
+  [DOI:10.3847/1538-3881/aabc4f](https://iopscience.iop.org/article/10.3847/1538-3881/aabc4f)
```
Comment (astrojuanlu) [inline docs/source/bibliography.md:53]: The DOI is correct, but the title of the paper changed.

**Review by astrojuanlu — CHANGES_REQUESTED** [review]
Thanks! I requested 1 change

File: docs/source/bibliography.md (hunk 1 above)
Comment (itsabhianant) [reply docs/source/bibliography.md:53]: I can't get where the title is on the website. Is this the title?

```
The Astropy Project: Building an Open-science Project and Status of the v2.0 Core Package*
```

Comment (itsabhianant) [reply docs/source/bibliography.md:53]: Or I have to give this title


The Astropy Collaboration, A. M. Price-Whelan, B. M. Sipőcz, H. M. Günther, P. L. Lim, S. M. Crawford, S. Conseil, D. L. Shupe, M. W. Craig, N. Dencheva "The Astropy Project: Building an Open-science Project and Status of the v2.0 Core Package"  *The Astropy Collaboration et al  AJ 156 123*(2018)

Comment (astrojuanlu) [reply docs/source/bibliography.md:53]: The second one

**Comment by itsabhianant** [comment]
Hi @astrojuanlu,

I have made the changes as per your instruction.

**Comment by astrojuanlu** [comment]
Thanks. There are two merge commits and I'd prefer not to merge them. If you could rebase, squash everything into one commit, and force push, I'd appreciate it .

**Comment by itsabhianant** [comment]
Should I close this PR and make a new PR with  a single commit?

**Comment by DhruvSondhi** [comment]
If you could rebase & squash that would be better than opening a new PR altogether

**Comment by itsabhianant** [comment]
Hi @astrojuanlu & @DhruvSondhi,

I rebased and squashed the commits. Hope now this is good to merge but please let me know if any more changes need to be made.

Best, 
Abhishek

**Review by astrojuanlu — APPROVED** [review]

**Comment by astrojuanlu** [comment]
Looks like the rebase was not complete :) But the squash was. Won't bother you more with this. Thanks!

**Pull request merged**